src/models/application.model.js

```diff
@@ -11,6 +11,8 @@ module.exports = class Application extends BaseModel {
     super()
     if (application) {
       this.accountId = application.accountId
+      // The following delay is required by the untilComplete method
+      this.delay = 250
```

Comment:
I would prefer this to be a constant, declared outside the class on line 9, rather than a this.delay property of the class.

i.e. const UPDATE_DELAY = 250

If we eventually decide this has to stay then we will probably want to move it to constants.js instead.

Either way, I don't think that it should be a property of the class.

Reply:
As discussed, using a constant will not allow the tests to override this value and therefore break the tests.  I have created the ticket https://eaflood.atlassian.net/browse/WE-801 to try and address this problem at a later date.